index.html

```diff
@@ -0,0 +1,231 @@
+        professional chef, start a new career, or just expand your cooking
+        skills.
+      </p>
+      <div class="main-time">
+        <h3 class="main-sub-heading">Registration starts 2022.10.15(MON)</h3>
+        <a class="zoom" href="#">Register now for Zoom link</a>
+      </div>
+    </section>
+
+    <section class="main-section2">
+      <img src="chef.jpg" alt="chef" class="chef-img" />
+      <h2 class="headline">Welcome to the Institute of</h2>
+      <h3 class="hl2">Culinary Education</h3>
+      <h4 class="hl3">Ranked No.1 Culinary School in South America*</h4>
+
+      <p class="para2">
+        The Institute of Culinary Education is a world-class culinary school
+        located in the heart of Santiago, Chile. We offer a variety of programs
+        to help you achieve your culinary goals, whether you’re looking to
+        become a professional chef, start a new career, or just expand your
+        cooking skills.
+      </p>
+      <div class="main-time">
+        <h3 class="main-sub-heading">Registration starts 2022.10.15(MON)</h3>
+        <a class="zoom" href="#">Register now for Zoom link</a>
+      </div>
+    </section>
+
+    <section id="program" class="middle-section">
+      <h2 class="program">The Program</h2>
+
+      <div class="middle-section-container">
+        <div class="middle-section-container2-1">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">Professional Chef Program</h3>
+          <p class="middle-section-para">
+            The Professional Chef Program is a 12-month, full-time program that
+            will teach you the culinary skills and business knowledge you need
+            to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-2">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Pastry Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Pastry Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-3">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Culinary Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Culinary Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+      </div>
+
+      <button type="button" class="btn">See the full program</button>
+    </section>
```

Comment:
- [ ] [OPTIONAL]  Good job so far 👍 Program cards should be 5 as shown in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and required design, but your cards are not totally! Please `style` them as in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), below I have tried to highlight the differences between you and Behance design!

![image](https://user-images.githubusercontent.com/63915024/191832560-eda0a475-eea6-415a-b597-d587925187cb.png)